src/style.css

```diff
@@ -2,0 +1,67 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.search-bar {
+  margin-top: 20px;
+}
+
+.search-bar-top {
+  height: 70px;
+  margin-top: 20px;
+}
+
+.search-bar-bottom {
+  height: 42px;
+  margin-top: 20px;
+}
+
+.search-bar__input {
+  box-sizing: border-box;
+  width: 100%;
+  height: 100%;
+  background-repeat: no-repeat;
+  background-image: url(./images/Search.svg);
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+
+  font-family: Avenir, sans-serif;
```

Comment:
You've declared the 'Avenir' font here, but there's no fallback font. It's a good practice to always include a fallback font, in case the custom font doesn't load. For example, you can use a generic font family like 'sans-serif' as a fallback.